.travis.yml

```diff
@@ -9,10 +10,11 @@ addons:
     - ubuntu-toolchain-r-test
     packages:
     - g++-5
+    - clang
 
 install:
 - export CXX="g++-5"
-- script/configure
+- scan-build script/configure
```

Comment:
Without running `./script/configure` under `scan-build` I get different results on my setup and travis. On my setup (with some extra debugging) `ccc-analyzer` is invoked correctly:

```
$ CC=`which gcc` CXX=`which g++` ./script/test -b
...
+ scan-build --use-analyzer=/usr/local/bin/clang --use-cc=/usr/bin/gcc --use-c++=/usr/bin/g++ --status-bugs make -j2 tests
scan-build: Using '/usr/local/bin/clang' for static analysis
  /usr/local/Cellar/scan-build/279/bin/../libexec/ccc-analyzer '-DTREE_SITTER_WRAP_MALLOC=true' -Iinclude -Isrc -Iexternals/utf8proc  -fasm-blocks -mpascal-strings -O0 -gdwarf-2 -arch x86_64 -Wall -Wextra -Wno-unused-parameter  -MMD -MF out/Test/.deps/out/Test/obj.target/runtime/src/runtime/document.o.d.raw  -c -o out/Test/obj.target/runtime/src/runtime/document.o src/runtime/document.c
  /usr/local/Cellar/scan-build/279/bin/../libexec/ccc-analyzer '-DTREE_SITTER_WRAP_MALLOC=true' -Iinclude -Isrc -Iexternals/utf8proc  -fasm-blocks -mpascal-strings -O0 -gdwarf-2 -arch x86_64 -Wall -Wextra -Wno-unused-parameter  -MMD -MF out/Test/.deps/out/Test/obj.target/runtime/src/runtime/error_costs.o.d.raw  -c -o out/Test/obj.target/runtime/src/runtime/error_costs.o src/runtime/error_costs.c
```
but the analyzer isn't invoked on travis:
```
+scan-build --use-analyzer=/usr/local/clang-3.5.0/bin/clang --use-cc=gcc --use-c++=g++-5 --status-bugs make -j2 tests
scan-build: Using '/usr/local/clang-3.5.0/bin/clang' for static analysis
  gcc '-DTREE_SITTER_WRAP_MALLOC=true' -Iinclude -Isrc -Iexternals/utf8proc  -Wall -Wextra -Wno-unused-parameter -g -std=c99 -MMD -MF out/Test/.deps/out/Test/obj.target/runtime/src/runtime/document.o.d.raw  -c -o out/Test/obj.target/runtime/src/runtime/document.o src/runtime/document.c
  gcc '-DTREE_SITTER_WRAP_MALLOC=true' -Iinclude -Isrc -Iexternals/utf8proc  -Wall -Wextra -Wno-unused-parameter -g -std=c99 -MMD -MF out/Test/.deps/out/Test/obj.target/runtime/src/runtime/error_costs.o.d.raw  -c -o out/Test/obj.target/runtime/src/runtime/error_costs.o src/runtime/error_costs.c
```

which is annoying. I assume this is a difference between the scan-build versions(?)